lib/ecto/migration.ex

```diff
@@ -1195,7 +1195,7 @@ defmodule Ecto.Migration do
   end
 
   def rename(%Index{} = current_index, to: new_name) do
-    Runner.execute({:rename, current_index, new_name})
+    Runner.execute({:rename, __prefix__(current_index), new_name})
```

Comment:
```suggestion
    Runner.execute({:rename, __prefix__(current_index), __prefix__(new_name)})
```
I think you want to do this so it can be reversible